src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1068,21 +1068,54 @@ private async Task SendInviteAsync(OrganizationUser orgUser, Organization organi
             IUserService userService)
         {
             var orgUser = await _organizationUserRepository.GetByIdAsync(organizationUserId);
+            
             if (orgUser == null)
             {
                 throw new BadRequestException("User invalid.");
             }
-
-            if (orgUser.Status != OrganizationUserStatusType.Invited)
+            
```

Comment:
Remove extra white-space

Reply:
I adjusted some of the white-spaces mentioned, but I personally like some space between unrelated conditionals: will gladly remove all of them though if that's what we'd like!

Reply:
Nope, that's perfect, wasn't an issue between conditionals, just the empty spaces on the blank line (`"______\n"` vs. just the newline, `"\n"`).